core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -821,6 +821,13 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
           OperandTypes.CHARACTER,
           SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {SPARK})
```

Comment:
could you just do `= SOUNDEX.withName('SOUNDEX_SPARK')` (or whatever name you choose)

Reply:
good point. updated it.